src/python/pants/java/nailgun_executor.py

```diff
@@ -183,6 +183,7 @@ def _await_socket(self, timeout):
     """Blocks for the nailgun subprocess to bind and emit a listening port in the nailgun stdout."""
```

Comment:
Hm. @kwlzn : I thought that we had added a custom nailgun protocol message for pids? It's interesting that this is arriving on stdout (...or perhaps this is "all socket output" instead?)

Reply:
this is the actual stdout stream from the nailgun java server subprocess invoke, so isn't going over the nailgun protocol afaik.

Reply:
ooh. K.